packages/core/src/browser/keybinding.spec.ts

```diff
@@ -173,6 +173,109 @@ describe('keybindings', () => {
         }
     });
 
+    it('should unregister all keybindings from a specific command', () => {
+        const otherKeybinding: Keybinding = {
+            command: TEST_COMMAND.id,
+            keybinding: 'F4'
+        };
+        keybindingRegistry.registerKeybinding(otherKeybinding);
+        const numOfKeybindingsForOtherCommand = keybindingRegistry.getKeybindingsForCommand(TEST_COMMAND.id).length;
+
+        const keybinding: Keybinding = {
+            command: TEST_COMMAND2.id,
+            keybinding: 'F5'
+        };
+        const keybinding2: Keybinding = {
+            command: TEST_COMMAND2.id,
+            keybinding: 'F6'
+        };
+
+        keybindingRegistry.registerKeybinding(keybinding);
+        keybindingRegistry.registerKeybinding(keybinding2);
+
+        keybindingRegistry.unregisterKeybinding(TEST_COMMAND2);
+
+        const bindings = keybindingRegistry.getKeybindingsForCommand(TEST_COMMAND2.id);
+        if (bindings) {
```

Comment:
Why not `expect`? Same the other `if`.

Reply:
Not really the thing we test here, but I will adapt that.